src/System.Numerics.Tensors/System/Numerics/Tensor.cs

```diff
@@ -1039,6 +1042,126 @@ void IList.RemoveAt(int index)
         }
         #endregion
 
+        #region IEnumerable<T> members
+        public IEnumerator<T> GetEnumerator()
```

Comment:
> As I mentioned above, I am not sure what the scenarios for enumerating tensors are. I can see how IEnumerable is the only interface that might make sense. 

Can this be explicit then
```csharp
IEnumerator<T> IEnumerable<T>.GetEnumerator()
```
So if it was determined in the future that a struct enumerator would be advantageous (no allocations; no virtual calls, inlining) - it wouldn't be a breaking change by changing the return type

Reply:
I will make it explicit.  I've also opened #1893 to consider adding the struct Enumerator.